tests/test_ingv_rcmt_reader_csv.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+import os.path
+import numpy
+from csep.utils import readers
+from csep.core import catalogs
+
+
+def get_datadir():
+    root_dir = os.path.dirname(os.path.abspath(__file__))
+    data_dir = os.path.join(root_dir, 'artifacts', 'Testing', 'ingv_rcmt-catalogs')
+    return data_dir
+
+class TestReadCatalog(unittest.TestCase):
+
+
+    def test_read_short_cat(self):
+
+        datadir = get_datadir()
+        file_ = 'ingv_rcmt-short.csv'
+        catalog_tuples = readers.read_ingv_rcmt_csv(os.path.join(datadir,file_))
+        self.assertEqual(len(catalog_tuples),10)
+
+    def test_read_large_cat(self):
+
+        datadir = get_datadir()
+        file_ = 'ingv_rcmt-large.csv'
+        catalog_tuples = readers.read_ingv_rcmt_csv(os.path.join(datadir,file_))
+        self.assertEqual(len(catalog_tuples),2340)
+        self.assertEqual(catalog_tuples[891][6],77.)
```

Comment:
i think it would be best to do a line by line. you can also use numpy.testing.assert_allclose() to compare floating point numbers.

Reply:
Ok!  This is my first 'formal' testing of code (always before it was hardtesting). So any suggestion of how to proceed from now on, are super appreciated.

Reply:
sounds good. i think for this test on first order we want to make sure that the data are read in correctly. bonus points if we can handle bad files.